partner_firstname/models/res_partner.py

```diff
@@ -94,11 +94,10 @@ def _get_names_order(self):
             'partner_names_order', self._names_order_default())
 
     @api.model
-    def _get_computed_name(self, lastname, firstname):
+    def _get_computed_name(self, lastname, firstname, order):
```

Comment:
To ensure compatibility with other modules calling _get_computed_name (like we have in the Dutch localization, but I assume there will be others), make order a keyword argument: order=None.

Then do not remove the assignment to order but make it: order = order if order else self._get_names_order() in case order was not passed into the method.